### DISCUSSION TITLE
Proposal: Add JSON format button

### DISCUSSION THREAD

**Discussion by WTIFS** [apolloconfig/apollo#5285] [Ideas]
Hi there, I'm planning to add a **JSON format** button under the configuration value textarea. As some of our colleagues use compact JSON format, and it's hard to see the differences when publishing.

The button will be looked as follows. Do you think it's a good idea?

| Scene | before | after |
| -- | -- | -- |
| Text Mode | <img width="800" alt="image" src="https://github.com/user-attachments/assets/348bae01-7ead-42fb-95cc-bbb3c0c30030">|<img width="800" alt="image" src="https://github.com/user-attachments/assets/cef63129-79c5-414e-ac81-21fa6a31ed7e">|
| Properties Mode | <img width="800" alt="image" src="https://github.com/user-attachments/assets/811d963d-7cef-4b4d-9090-90e7e7ac5a6a"> | <img width="800" alt="image" src="https://github.com/user-attachments/assets/146df723-b785-462d-b695-aba4ad3e6783">|

**Comment by nobodyiam** [comment]
This feature is interesting.

**Comment by hezhangjian** [comment]
Great idea!
